qa-niaid.planx-pla.net/portal/gitops.json

```diff
@@ -97,13 +97,23 @@
             "items": [
               {
                 "type": "pdf",
-                "link": "https://qa-niaid.planx-pla.net/dashboard/Public/files/2020-1138%20v200917FINALDUA.pdf",
+                "link": "qa-niaid.planx-pla.net/dashboard/Public/files/NIAIDDUA2021Accessclinicaldata@NIAID.pdf",
                 "name": "Data Use Agreement (DUA)"
               },
               {
                 "type": "pdf",
                 "link": "https://qa-niaid.planx-pla.net/dashboard/Public/files/NIAID%20Data%20Access%20Request%20FormSept2020.pdf",
                 "name": "Data Access Request (DAR)"
+              },
+              {
+                "type": "pdf",
+                "link": "qa-niaid.planx-pla.net/dashboard/Public/files/ACTT-1 Data Use Limitations.pdf",
+                "name": "ACTT-1 Data Use Limitations"
+              },
+              {
+                "type": "pdf",
+                "link": "qa-niaid.planx-pla.net/dashboard/Public/files/ACTT-2 Data Use Limitations.pdf",
+                "name": "ACTT-2 Data Use Limitations"
```

Comment:
mm the ACTT-1 and ACTT-2 files will both be displayed on all the study pages...

Reply:
I think we have no other option, right?